app/src/main/res/values/strings.xml

```diff
@@ -311,6 +311,10 @@
     <string name="dialog_action_logout">Logout</string>
     <string name="dialog_title_confirm_deletion">Confirm deletion</string>
     <string name="dialog_title_confirm_logout">Confirm logout</string>
+    <string name="dialog_title_confirm_back">Are you sure you want to go back?</string>
+    <string name="dialog_message_confirmation_back">All unsaved changes will be lost.</string>
+    <string name="dialog_action_go_back">Go back</string>
+    <string name="dialog_action_stay">Stay here</string>
```

Comment:
I think it would be much better and simple if we name our actions as `Yes` or `Cancel` because we are asking the user a Yes/No question (`Are you sure you want to go back?`)